components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/ConfigStoreFunctionalityLockPropertyHandler.java

```diff
@@ -68,7 +69,8 @@ public static ConfigStoreFunctionalityLockPropertyHandler getInstance() {
                                             functionalityIdentifier);
                     properties = new ResourceToProperties().apply(resource);
                 } else {
-                    throw new UnsupportedOperationException("User Functionality properties are not configured.");
+                    log.debug("User Functionality properties are not configured. Resorting to default values.");
```

Comment:
let's check here whether the debug logs are enabled

Reply:
Resolved with https://github.com/wso2-extensions/identity-governance/pull/455/commits/464cc57330f40baad5061fd4190f14ea4244955b.